src/impl/peerconnection.cpp

```diff
@@ -1247,4 +1247,11 @@ void PeerConnection::updateTrackSsrcCache(const Description &description) {
 		    description.media(i));
 }
 
+void PeerConnection::setConfiguration(const Configuration& config_) {
```

Comment:
I can merge the suggestion, but what benefits do you expect the move configuration? Passing by value is going to create a new copy of the structs, and as the struct does not store any resource that is costly to acquire/release (like a std::shared_ptr), the move will behave as another copy. It will probably be optimized out by the encoder, but at most it will be the same as passing by const ref and doing a copy later.

Reply:
As you say it's not a matter of performance here, just consistency with how configuration structs are handled in the codebase, and in general I like passing objects the function stores by value, and parameters the function only accesses by reference. Thank you for merging the suggestion.

Reply:
If you are fine, I am fine :)